vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_authorization.cc

```diff
@@ -41,7 +42,7 @@ void UpholdAuthorization::Authorize(
   const auto current_one_time = wallet->one_time_string;
 
   // we need to generate new string as soon as authorization is triggered
-  wallet->one_time_string = GenerateRandomString(ledger::is_testing);
+  wallet->one_time_string = util::GenerateRandomHexString();
```

Comment:
@emerick Do we need to regenerate the `code_verifier` in this routine as well? (I'm assuming that it should be generated uniquely for each request.)

Reply:
Great catch, thanks. I'll add that to the `rewards-bitflyer-integration` branch now.

Reply:
Done: https://github.com/brave/brave-core/commit/ae0bd3c978cc2758653ed27bc33ae3c12df38625